Sources/SwiftyMonaco/MonacoViewController.swift

```diff
@@ -118,6 +143,8 @@ public class MonacoViewController: ViewController, WKUIDelegate, WKNavigationDel
     }
```

Comment:
Evaluation is moved to a separate function because it's being called from multiple places